src/Monolog/ResettableInterface.php

```diff
@@ -27,5 +27,10 @@
  */
 interface ResettableInterface
 {
+    /**
+     * Function that executes the resetting of a Processor
```

Comment:
This isn't Processor specific, and really this whole sentence adds no value, I'd remove it.

Reply:
Removed.